<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataAnalysis</class>
 <widget class="QMainWindow" name="DataAnalysis">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1148</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataAnalysis</string>
  </property>
  <property name="windowIcon">
   <iconset resource="xiaoniao.qrc">
    <normaloff>:/xiaoniao/myico.ico</normaloff>:/xiaoniao/myico.ico</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Arial&quot;;</string>
      </property>
      <property name="title">
       <string>Info</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; color:#7d7d7d;&quot;&gt;Pulish by Will at 202105,Coprright 2021&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextBrowserInteraction</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
      </property>
      <property name="title">
       <string>Main </string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTabWidget" name="DataSource">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Database Generate</string>
          </attribute>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>1071</width>
             <height>531</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="3">
               <widget class="QToolButton" name="pushButtonDataSo">
                <property name="text">
                 <string>SetPath</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_4">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Map File</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLineEdit" name="lineEditDataSource">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>This is a tool tip</string>
                </property>
                <property name="statusTip">
                 <string>This is a status tip</string>
                </property>
                <property name="whatsThis">
                 <string>This is &quot;what is this&quot;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>DB Path</string>
                </property>
               </widget>
              </item>
              <item row="4" column="3">
               <widget class="QToolButton" name="pushButtonDBPath">
                <property name="text">
                 <string>SetPath</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>DataSource</string>
                </property>
               </widget>
              </item>
              <item row="4" column="2">
               <widget class="QLineEdit" name="lineEditDBPath">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>This is a tool tip</string>
                </property>
                <property name="statusTip">
                 <string>This is a status tip</string>
                </property>
                <property name="whatsThis">
                 <string>This is &quot;what is this&quot;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QToolButton" name="pushButtonMapFile">
                <property name="text">
                 <string>SetPath</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLineEdit" name="lineEditMapFile">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>This is a tool tip</string>
                </property>
                <property name="statusTip">
                 <string>This is a status tip</string>
                </property>
                <property name="whatsThis">
                 <string>This is &quot;what is this&quot;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="pushButtonStartD">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 20pt &quot;Agency FB&quot;;</string>
                </property>
                <property name="text">
                 <string>Start</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonStopD">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 20pt &quot;Agency FB&quot;;</string>
                </property>
                <property name="text">
                 <string>Stop</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonClearD">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 20pt &quot;Agency FB&quot;;</string>
                </property>
                <property name="text">
                 <string>Clear</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTextBrowser" name="textBrowserD">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_7">
          <attribute name="title">
           <string>Plot Generate</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="1" column="0">
              <widget class="QLabel" name="label_6">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Title File</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>DB File</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEditDBFile">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>This is a tool tip</string>
               </property>
               <property name="statusTip">
                <string>This is a status tip</string>
               </property>
               <property name="whatsThis">
                <string>This is &quot;what is this&quot;</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QToolButton" name="pushButtonDBFile">
               <property name="text">
                <string>SetPath</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEditTitleFile">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>This is a tool tip</string>
               </property>
               <property name="statusTip">
                <string>This is a status tip</string>
               </property>
               <property name="whatsThis">
                <string>This is &quot;what is this&quot;</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QToolButton" name="pushButtonTitleFile">
               <property name="text">
                <string>SetPath</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="pushButtonStartP">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 20pt &quot;Agency FB&quot;;</string>
               </property>
               <property name="text">
                <string>Start</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonStopP">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 20pt &quot;Agency FB&quot;;</string>
               </property>
               <property name="text">
                <string>Stop</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonClearP">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 20pt &quot;Agency FB&quot;;</string>
               </property>
               <property name="text">
                <string>Clear</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTextBrowser" name="textBrowserP"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1148</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionDarkStyle"/>
    <addaction name="actionLightStyle"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About DataAnalysis</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuHelp"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBarMenus">
   <property name="windowTitle">
    <string>Toolbar with menu</string>
   </property>
   <property name="toolTip">
    <string>Toolbar with menu</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionActionA">
   <property name="text">
    <string>Action A</string>
   </property>
  </action>
  <action name="actionActionSubA">
   <property name="text">
    <string>Action A Sub</string>
   </property>
   <property name="toolTip">
    <string>Action A Sub</string>
   </property>
  </action>
  <action name="actionActionSubB">
   <property name="text">
    <string>Action B Sub</string>
   </property>
  </action>
  <action name="actionActionDelayedA">
   <property name="text">
    <string>Action Delayed A</string>
   </property>
   <property name="toolTip">
    <string>Action Delayed A</string>
   </property>
  </action>
  <action name="actionActionDelayedSubA">
   <property name="text">
    <string>Action Delayed Sub A</string>
   </property>
   <property name="toolTip">
    <string>Action Delayed Sub A</string>
   </property>
  </action>
  <action name="actionActionCheckableA">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Action Checkable A</string>
   </property>
   <property name="toolTip">
    <string>Action Checkable A</string>
   </property>
  </action>
  <action name="actionActionCheckableSubAChecked">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Action Checkable Sub A Checked</string>
   </property>
   <property name="toolTip">
    <string>Action Checkable Sub A Checked</string>
   </property>
  </action>
  <action name="actionActionCheckableSubAUnchecked">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Action Checkable Sub A Unchecked</string>
   </property>
   <property name="toolTip">
    <string>Action Checkable Sub A Unchecked</string>
   </property>
  </action>
  <action name="actionNewB">
   <property name="text">
    <string>New B</string>
   </property>
  </action>
  <action name="actionNewC">
   <property name="text">
    <string>New C</string>
   </property>
  </action>
  <action name="actionNewD">
   <property name="text">
    <string>New D</string>
   </property>
  </action>
  <action name="actionNewE">
   <property name="text">
    <string>New E</string>
   </property>
  </action>
  <action name="actionActionIconADis">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../¤Treasure/☆Collection/python_pkg/QDarkStyleSheet-master/qdarkstyle/dark/style.qrc">
     <normaloff>:/qss_icons/dark/rc/window_undock_focus.png</normaloff>:/qss_icons/dark/rc/window_undock_focus.png</iconset>
   </property>
   <property name="text">
    <string>Action Icon A</string>
   </property>
  </action>
  <action name="actionDarkStyle">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../¤Treasure/☆Collection/python_pkg/QDarkStyleSheet-master/qdarkstyle/dark/style.qrc">
     <normaloff>:/qss_icons/dark/rc/window_close_focus.png</normaloff>:/qss_icons/dark/rc/window_close_focus.png</iconset>
   </property>
   <property name="text">
    <string>DarkStyle</string>
   </property>
   <property name="toolTip">
    <string>Action Icon B</string>
   </property>
  </action>
  <action name="actionLightStyle">
   <property name="icon">
    <iconset resource="../../../¤Treasure/☆Collection/python_pkg/QDarkStyleSheet-master/qdarkstyle/dark/style.qrc">
     <normaloff>:/qss_icons/dark/rc/window_grip_focus.png</normaloff>:/qss_icons/dark/rc/window_grip_focus.png</iconset>
   </property>
   <property name="text">
    <string>LightStyle</string>
   </property>
  </action>
  <action name="actionActionDis">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Action Disabled</string>
   </property>
   <property name="toolTip">
    <string>Action Disabled</string>
   </property>
  </action>
  <action name="actionActionCheckableCheckedDis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Action Checkable Checked Disabled</string>
   </property>
   <property name="toolTip">
    <string>Action Checkable Checked Disabled</string>
   </property>
  </action>
  <action name="actionActionCheckableUncheckedDis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Action Checkable Unchecked Disabled</string>
   </property>
   <property name="toolTip">
    <string>Action Checkable Unchecked Disabled</string>
   </property>
  </action>
  <action name="actionActionTranspIcon">
   <property name="icon">
    <iconset resource="../../../¤Treasure/☆Collection/python_pkg/QDarkStyleSheet-master/qdarkstyle/dark/style.qrc">
     <normaloff>:/qss_icons/dark/rc/transparent.png</normaloff>:/qss_icons/dark/rc/transparent.png</iconset>
   </property>
   <property name="text">
    <string>Action Transp. Icon</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../¤Treasure/☆Collection/python_pkg/QDarkStyleSheet-master/qdarkstyle/dark/style.qrc"/>
  <include location="xiaoniao.qrc"/>
 </resources>
 <connections/>
</ui>
